sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -29,8 +31,6 @@
 
 #define MAX_NO_PROPS 2
 
-extern struct bus_type sdw_bus_type;
-
```

Comment:
can you move this to a different patch, it makes my life easier when I squash patches...

Reply:
yes, done!